docs/zh/client/other-language-client-user-guide.md

```diff
@@ -2,10 +2,10 @@
 
 另外，如果有团队/个人有兴趣的话，也欢迎帮助我们来实现其它语言的客户端，具体细节可以联系@nobodyiam和@lepdou。
 
->注：目前已有热心用户贡献了Go、Python、NodeJS、PHP、C++的客户端，更多信息可以参考[Go、Python、NodeJS、PHP等客户端使用指南](zh/usage/third-party-sdks-user-guide)
+>注：目前已有热心用户贡献了Go、Python、NodeJS、PHP、C++的客户端，更多信息可以参考[Go、Python、NodeJS、PHP等客户端使用指南](zh/client/third-party-sdks-user-guide)
```

Comment:
The `/zh/client/third-party-sdks-user-guide` was removed.

Reply:
Yes, the `/en/client/third-party-sdks-user-guide` will be removed as well.

Reply:
This link will be replace with prue text.